README.md

```diff
@@ -35,6 +35,7 @@ Feel free to make an issue to discuss amending the scope.
 
 - The [PureScript Book](https://leanpub.com/purescript/read) is the recommended approach to learning the language, since it covers more material in greater depth. However, it is not updated yet for the 0.12 version of the compiler.
 - [Language Reference](language/README.md)
+- [PureScript: Jordan's Reference](https://github.com/JordanMartinez/purescript-jordans-reference): An up-to-date project covering Getting Started, Build Tools, PureScript's verified-by-compilation syntax via meta-language, FP design patterns, and PureScript's ecosystem.
```

Comment:
What is "verified-by-compilation syntax via meta-language"? I'd like to use more common words for this idea, if possible.

Reply:
:sweat_smile: Yeah, that's a good point.

Basically "meta-language" means I use the name of the actual terms to explain what they represent:
```purescript
-- instead of:
f :: Int -> String
f x = show x

-- I write:
function :: ArgType -> ReturnType
function argValue = show argValue

-- and again
class TypeClass someType where
  someFunction :: someType -> String

-- needed to compile
type ArgType = Int
type ReturnType = String
```
"Verified-by-compilation" means the above syntax is actual Purescript code that compiles. So, rather than me saying "Hey! I claim that this is correct; just trust me!", I can say "Hey, I can prove that this works; no need to trust me." (Obviously, there's no CI that backs up that claim yet, but it's on my todo list).

Reply:
As an example of the meta-language, see [how I explained Pattern Matching](https://github.com/JordanMartinez/purescript-jordans-reference/blob/latestRelease/11-Syntax/01-Basic-Syntax/src/02-Data-and-Functions/05-Pattern-Matching-in-Functions.purs)

Reply:
Hmm, I would just say "...covering "Getting Started", build tools, syntax with examples, FP design patterns, and the ecosystem. If you decide to change how you explain it, or what you call it, we wouldn't have to make a patch to this purescript/documentation project, also.
What do you think? Do you feel strongly about your original choice of words?

Reply:
Up here, I'd like this language to be more simple.